ngraph/frontend/onnx_import/src/utils/onnx_test_util.hpp

```diff
@@ -0,0 +1,54 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <string>
+
+#include "onnx_import/utils/onnx_importer_visibility.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        struct ONNX_IMPORTER_API ComparisonResult
```

Comment:
Looks like we don't need to export this struct.

Reply:
For now we have to so that it can be used in the editor tests.